pkgs/development/interpreters/wasm3/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, cmake, fetchFromGitHub, cacert, git }:
+
+stdenv.mkDerivation rec {
+  pname = "wasm3";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "wasm3";
+    repo = "wasm3";
+    rev = "v${version}";
+    sha256 = "07zzmk776j8ydyxhrnnjiscbhhmz182a62r6aix6kfk5kq2cwia2";
+  };
+
+  # https://github.com/vshymanskyy/uvwasi is fetched by the build script.
```

Comment:
The `uvwasi` source should be declared. We can disable it with `-DBUILD_WASI=simple` flag. `nodejs` already bundles it.

Reply:
For while I just added `-DBUILD_WASI=simple`. In the future we can add an option to build with `uvwasi`.